### DISCUSSION TITLE
https://docs.shouldly.io not reachable

### DISCUSSION THREAD

**Discussion by whobday** [shouldly/shouldly#858] [General] [answered]
I'm not able to reach https://docs.shouldly.io.  I'm not sure when this started happening, but it's been down for at least a month.

**Comment by themanuelflores** [comment]
Looks like the main domain is reserved by a domain registrar, NameCheap

**Comment by jnm2** [comment]
Thanks for reporting this. I asked for help from those who might own the domain in https://github.com/shouldly/shouldly/issues/860.

**Comment by jnm2** [comment, marked as answer]
The `shouldly.io` domain was lost. The docs are now online at <https://docs.shouldly.org>.
